src/rust/dbv2/src/schema.rs

```diff
@@ -1,24 +1,640 @@
 // @generated automatically by Diesel CLI.
 
 diesel::table! {
-    market_registration_events (txn_version, event_idx) {
+    account_transactions (account_address, transaction_version) {
+        transaction_version -> Int8,
+        #[max_length = 66]
+        account_address -> Varchar,
+        inserted_at -> Timestamp,
+    }
+}
+
+diesel::table! {
+    ans_lookup (transaction_version, write_set_change_index) {
+        transaction_version -> Int8,
+        write_set_change_index -> Int8,
+        #[max_length = 64]
+        domain -> Varchar,
+        #[max_length = 64]
+        subdomain -> Varchar,
+        #[max_length = 66]
+        registered_address -> Nullable<Varchar>,
+        expiration_timestamp -> Nullable<Timestamp>,
+        #[max_length = 140]
+        token_name -> Varchar,
+        is_deleted -> Bool,
+        inserted_at -> Timestamp,
+    }
+}
```

Comment:
A lot of these tables are irrelevant for us, for example Aptos Name Service (ANS) tables

If these are included in here due to deisel autogen, is there anyway to exclude them?